web/angular-wallet/src/app/app.component.ts

```diff
@@ -72,7 +72,6 @@ export class AppComponent extends UnsubscribeOnDestroy implements OnInit, OnDest
       .subscribe((ready) => {
         if (ready) {
           this.updateAccounts();
-          this.checkBlockchainStatus();
```

Comment:
that caused the issue

Reply:
aye, I believe this will also result in a user seeing an outdated balance for a pretty long period of time whenever they first load the wallet. Maybe this needs to be wrapped in a setTimeout or something?

Reply:
I'll double check this... in worst case we use a timeout again